File: src/main/java/me/tigermouthbear/nebulous/modifiers/ClassNameModifier.java
package me.tigermouthbear.nebulous.modifiers;

import me.tigermouthbear.nebulous.util.RandomString;

import java.util.HashMap;
import java.util.Map;

public class ClassNameModifier extends Modifier
{
	public void modify()
	{
		Map<String, String> remap = new HashMap<>();

		getClasses().forEach(classNode ->
				remap.put(classNode.name, "AresClientOnTop" + RandomString.genRandomString()));

		applyRemap(remap);
	}
}

File: src/main/java/me/tigermouthbear/nebulous/modifiers/ClassNameModifier.kt
package me.tigermouthbear.nebulous.modifiers

import me.tigermouthbear.nebulous.util.RandomString
import org.objectweb.asm.tree.ClassNode
import java.util.*

class ClassNameModifier : Modifier() {
    override fun modify() {
        val remap: MutableMap<String?, String?> = HashMap()
        classes.forEach { classNode: ClassNode -> remap[classNode.name] = RandomString.genRandomString() }
        applyRemap(remap)
    }
}